storage/store/inmemory/unsynchronized/events_test.go

```diff
@@ -56,3 +59,33 @@ func TestEvents_HappyPath(t *testing.T) {
 	assert.Contains(t, typeEvents, event1)
 	assert.Contains(t, typeEvents, event3)
 }
+
+func TestEvents_Persist(t *testing.T) {
+	dbtest.RunWithDB(t, func(t *testing.T, db storage.DB) {
+		eventStore := NewEvents()
+		block := unittest.BlockFixture()
+		transaction := unittest.TransactionFixture()
+		event := unittest.EventFixture(flow.EventAccountCreated, 0, 0, transaction.ID(), 200)
+		expectedStoredEvents := flow.EventsList{event}
+
+		// Store events
+		err := eventStore.Store(block.ID(), []flow.EventsList{expectedStoredEvents})
+		require.NoError(t, err)
+		require.NoError(t, db.WithReaderBatchWriter(func(rw storage.ReaderBatchWriter) error {
+			return eventStore.AddToBatch(rw)
+		}))
+
+		// Encode event key
+		eventCode := byte(102) // taken from operation/prefix.go
+		key := operation.EventPrefix(eventCode, block.ID(), event)
+
+		// Get event
+		reader := db.Reader()
+		value, closer, err := reader.Get(key)
+		defer closer.Close()
+		require.NoError(t, err)
+
+		// Ensure event with such a key was stored in DB
+		require.NotEmpty(t, value)
```

Comment:
instead of a direct db call, how about using the same operation that the storage objects use
https://github.com/onflow/flow-go/blob/0fbe3a21fa71fd0b91a9d441a71339b84cc16c54/storage/store/events.go#L22-L23


then you can check that the returned events match